requirements.txt

```diff
@@ -4,4 +4,5 @@ pyjwt==2.8.0
 Flask-HTTPAuth==4.8.0
 python-dotenv==1.0.0
 requests==2.31.0
-filelock==3.13.1
\ No newline at end of file
+filelock==3.13.1
+pre-commit
```

Comment:
pre-commit will rather go in `requirements-dev.txt` as it is a dev requirement not a core requirement.